### DISCUSSION TITLE
options.useFluidContainers

### DISCUSSION THREAD

**Discussion by blondie63** [Laravel-Backpack/community-forum#650] [Q&A (Help)] [answered]
On my last prj converted to BKP6 i've set config/backpack/ui.php with:
```
    'view_namespace' => 'backpack.theme-coreuiv2::',
    'view_namespace_fallback' => 'backpack.theme-coreuiv2::',
```

So i'm not using tabler theme, but looking in laraval.log i've found many of these:

```
[2023-08-25 10:01:17] local.ERROR: Could not find config key: options.useFluidContainers. Neither in the Backpack theme, nor in the fallback theme, nor in ui.
```

Some info ?

**Comment by pxpm** [comment, marked as answer]
Hey @blondie63 

Search in your project for `useFluidContainers`. In some blade file it must be using it. 

I've searched in coreuiv2 repository and CRUD and I didn't find any mention of it: https://github.com/search?q=repo%3ALaravel-Backpack%2Ftheme-coreuiv2+useFluidContainers&type=code

https://github.com/search?q=repo%3ALaravel-Backpack%2FCRUD+useFluidContainers&type=code

Let me know how it goes.

Cheers

**Reply by blondie63** [reply]
Just searched before this post and the only one entry is in theme-tabler folder..
I've both theme intalled

**Reply by pxpm** [reply]
Do you use Laravel DebugBar ? 

Can you check on the views tab, if you are loading any view from tabler namespace ? 

![image](https://github.com/Laravel-Backpack/community-forum/assets/7188159/40364550-af8a-41d8-a9af-74f0ada3b1a2)

Cheers

**Reply by blondie63** [reply]
I'm using Telescope, i'll try to search and let you know..

**Comment by blondie63** [comment]
@pxpm Hi, no views with key useFluidContainers or tabler !
<img width="1296" alt="pic-backend-telescope-tabler" src="https://github.com/Laravel-Backpack/community-forum/assets/1261851/df657b57-c620-4780-8c45-cb47fdda76fe">
<img width="1517" alt="pic-backend-telescope-useFluidContainers" src="https://github.com/Laravel-Backpack/community-forum/assets/1261851/5df957c4-21b3-4244-85dc-80b520ae822a">
